### PR TITLE
dalle3 add style consistency parameter

### PR DESCRIPTION
**Pull request by charli117**
# Description

dalle3 add style consistency parameter: seed id, Ensure consistency in series image generation

Fixes # (issue)

## Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update, included: [Dify Document](https://github.com/langgenius/dify-docs)
- [x] Improvement, including but not limited to code refactoring, performance optimization, and UI/UX improvement
- [ ] Dependency upgrade

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] TODO

# Suggested Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] I ran `dev/reformat`(backend) and `cd web && npx lint-staged`(frontend) to appease the lint gods
- [ ] `optional` I have made corresponding changes to the documentation 
- [ ] `optional` I have added tests that prove my fix is effective or that my feature works
- [ ] `optional` New and existing unit tests pass locally with my changes

### CODE REVIEW

File: api/core/tools/provider/builtin/dalle/tools/dalle3.py [hunk 1]
```diff
@@ -68,8 +73,15 @@ def _invoke(self,
         result = []
 
         for image in response.data:
-            result.append(self.create_blob_message(blob=b64decode(image.b64_json), 
-                                                   meta={ 'mime_type': 'image/png' },
-                                                    save_as=self.VARIABLE_KEY.IMAGE.value))
+            result.append(self.create_blob_message(blob=b64decode(image.b64_json),
+                                                   meta={'mime_type': 'image/png'},
+                                                   save_as=self.VARIABLE_KEY.IMAGE.value))
+        result.append(self.create_text_message(f'\nGenerate image source to Seed ID: {seed_id}'))
```
Comment (takatost) [inline api/core/tools/provider/builtin/dalle/tools/dalle3.py:79]: Is this code necessary?

**Review by takatost — CHANGES_REQUESTED** [review]

File: api/core/tools/provider/builtin/dalle/tools/dalle3.py (hunk 1 above)
Comment (charli117) [reply api/core/tools/provider/builtin/dalle/tools/dalle3.py:79]: @takatost necessary, to pass the seed id of the previous task, for example:
dell3 图像一致性保持

**Review by crazywoola — APPROVED** [review]

**Review by crazywoola — APPROVED** [review]

**Review by takatost — APPROVED** [review]
LGTM

**Comment by XiaoLey** [comment]
image

## OpenAI Instruction: 

image

image

Even though I haven't seen `seed` parameter in the official OpenAI instructions for `Create Image`, could you please verify it?

**Pull request merged**